### ISSUE TITLE
Remove the eomg job

### ISSUE THREAD

**Issue by DavidHuber-NOAA** [NOAA-EMC/global-workflow#2525]
### What new functionality do you need?

The eomg job is no longer supported for the GSI and the GDASApp has its own jobs to handle observational diagnostics (i.e. when `lobsdiag=".true."`): atmensanlobs and atmensanlsol.  Thus, the eomg job should be removed from the global workflow without affecting the new atmensanlobs and atmensanlsol jobs.

### What are the requirements for the new functionality?

All references to eomg are removed and job-specific scripts are removed from the global workflow without affecting the new GDASApp jobs.  Also, eomg-specific archiving is removed and/or replaced with GDASApp-specific archiving instructions.

**Comment by aerorahul** [comment]
@CatherineThomas-NOAA Is the functionality of `eomg.sh` and its use needed to be supported?

**Comment by CatherineThomas-NOAA** [comment]
@aerorahul I don't think so, especially given that we didn't notice the job was missing.  Let me ask around a bit though.

**Comment by aerorahul** [comment]
Thanks @CatherineThomas-NOAA.  In that case we will eliminate the configurable option for `lobsdiag_for_enkf` and simplify the scripting.

**Comment by RussTreadon-NOAA** [comment]
g-w issue #2415 was opened to add JEDI ATM equivalents to to `eomg` and `eobs`.  These jobs perform observation quality control and h(x) for ensemble member states, `eomg`, and ensemble mean states, `eobs`.   While we may not need `eomg.sh` for GSI ATM DA.  We will likely be adding it for JEDI ATM DA.

**Comment by DavidHuber-NOAA** [comment]
Adding `blocked` while waiting for the GDASApp-equivalent job to replace eomg.

**Comment by RussTreadon-NOAA** [comment]
While we do not have a separate eomg job for GDASApp, the functionality of eomg currently exists in g-w.  When GDASApp is run with `lobsdiag_forenkf=".false."`,   we run a single job, atmensanlletkf, which combines eomg and eupd into a single executation of `gdas.x`.  The wall time and memory requirements for this combined approach make it unpractical for all but coarse resolutions with few ensemble members.  

When GDASApp is run `lobsdiag_forenkf=".true."` we have two g-w jobs:  atmensanlobs and atmensanlsol.  Job atmensanlobs is the GDASApp equivalent of GSI-based eobs.  The split approach is more efficient in terms of memory and wall time.

**Comment by DavidHuber-NOAA** [comment]
OK, so then it sound like this issue is OBE.  Agreed @aerorahul @WalterKolczynski-NOAA?

**Comment by DavidHuber-NOAA** [comment]
I updated the description of this issue to reflect that the eomg job should be removed from the global workflow without affecting the new GDASApp equivalent jobs.  I am also removing the `blocked` label.

**Comment by DavidHuber-NOAA** [comment]
Resolved by #3331.
